Slambook/index.js

```diff
@@ -1,14 +1,26 @@
 let form;
 
 let allSlamArray=[];
+let formValue = {};
+let err ;
 //Write code below to select the form element
 form = document.querySelector('form')
 console.log(form,"form");
 // Write your code above
```

Comment:
The 'remove' method for elements does not take any arguments. Remove 'element' from the 'remove' method at line 9.